models/bpreg/dockerfiles/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM mhubai/base:latest
+
+LABEL authors="bspai@bwh.harvard.edu,lnuernberg@bwh.harvard.edu"
+
+# # clone mhub implementation
+ARG MHUB_MODELS_REPO
+
+RUN buildutils/import_mhub_model.sh bpreg ${MHUB_MODELS_REPO}
+
+# Install additional pip packages
+RUN pip3 install --upgrade pip
```

Comment:
- combine the three pip commands into one RUN block to reduce layer buildup
- consistent use op `pip3` / `pip`